armi/reactor/assemblies.py

```diff
@@ -1250,6 +1249,15 @@ def isOnWhichSymmetryLine(self):
         grid = self.parent.spatialGrid
         return grid.overlapsWhichSymmetryLine(self.spatialLocator.getCompleteIndices())
 
+    def orientBlocks(self, cornersUp):
+        """Add special grids to the blocks inside this Assembly, respecting their orientation."""
+        for b in self:
+            if b.spatialGrid is None:
+                try:
+                    b.autoCreateSpatialGrids(cornersUp)
+                except (ValueError, NotImplementedError) as e:
+                    runLog.warning(str(e), single=True)
+
```

Comment:
Should this live on the `HexAssembly` because we have a `cornerUp` flag that is hex specific?

Reply:
All assemblies have a `.autoCreateSpatialGrids()` method though, right?

Reply:
Blocks yes, but the `cornersUp` switch is hex specific. What does this mean for a cartesian grid? I get that it's good OOP practice to have the same signatures for overridden methods, but this feels in the vein of "everything is hexagonal in ARMI"